cl_sii/rcv/data_models.py

```diff
@@ -297,39 +340,63 @@ class RcNoIncluirDetalleEntry(RcRegistroDetalleEntry):
     RC_ESTADO_CONTABLE = RcEstadoContable.NO_INCLUIR
 
 
-@dataclasses.dataclass(frozen=True)
+@pydantic.dataclasses.dataclass(
+    frozen=True,
+    config=type('Config', (), dict(
+        arbitrary_types_allowed=True,
+    ))
+)
 class RcReclamadoDetalleEntry(RcvDetalleEntry):
 
     """
     Entry of the "detalle" of an RC ("Registro de Compras") / "reclamado".
     """
 
+    ###########################################################################
+    # constants
+    ###########################################################################
+
+    DATETIME_FIELDS_TZ = SII_OFFICIAL_TZ
+
+    ###########################################################################
+    # fields
+    ###########################################################################
+
     RCV_KIND = RcvKind.COMPRAS
     RC_ESTADO_CONTABLE = RcEstadoContable.RECLAMADO
 
-    emisor_razon_social: str = dc_field()
+    emisor_razon_social: str
     """
     "Razón social" (legal name) of the "emisor" of the "documento".
     """
 
     # TODO: docstring
     # note: must be timezone-aware.
-    fecha_reclamo_dt: Optional[datetime] = dc_field()
-
-    def __post_init__(self) -> None:
-        super().__post_init__()
-
-        if not isinstance(self.emisor_razon_social, str):
-            raise TypeError("Inappropriate type of 'emisor_razon_social'.")
-        cl_sii.dte.data_models.validate_contribuyente_razon_social(self.emisor_razon_social)
-
-        if self.fecha_reclamo_dt is not None:
-            if not isinstance(self.fecha_reclamo_dt, datetime):
-                raise TypeError("Inappropriate type of 'fecha_reclamo_dt'.")
-            tz_utils.validate_dt_tz(self.fecha_reclamo_dt, SII_OFFICIAL_TZ)
+    fecha_reclamo_dt: Optional[datetime] = None
```

Comment:
See review comment for field `RvDetalleEntry.fecha_acuse_dt`.

```suggestion
    fecha_reclamo_dt: Optional[datetime]
```